devfiles/src/classes/Panel.ts

```diff
@@ -41,32 +45,40 @@ export default class Panel {
     public dataFilterer: DataFilterer;
     public readonly uuid: number;
     public minHeight: number;
-    public lifetimeWidgetCount: number;
+    public lifetimeWidgetCount: Map<string, number> = new Map();
```

Comment:
If I add a map then delete it then add it again should it be map 2? I thought it would be better if we counted the number of widgets of the same type in the same panel above the new widget then used that count. So if I have a map and a table, new map would be named map 2, even if I delete it and re-add it

Reply:
It opens a can of worms (i.e. if I add 2 maps, delete the first one, then I have a map named "Map 2", and the next map needs to be aware of it)

Reply:
Ah ok